domain/src/main/java/org/commcare/dalvik/domain/model/LanguageManager.kt

```diff
@@ -0,0 +1,113 @@
+package org.commcare.dalvik.domain.model
+
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+
+object LanguageManager {
+
+    const val DEFAULT_TRANSLATIONS = "{\n" +
```

Comment:
As ABDM supports multiple language. So we  kept the DEFAULT TRANSLATION i.e ENGLISH in code itself.
Now as the application starts it fetches the translation from the server along with the language code provided from the CC application.
So in case of following scenarios the DEFAULT TRANSLATION present in the application is used.

1. Translation API fails
2. Any key that is not present in the translation received from server is picked from the DEFAULT TRANSLATION 

So we can say this is the backup for translation support

Reply:
But is not `DEFAULT_TRANSLATIONS` same as `en/language.json` ? If so, why not use the `.json` file directly as `DEFAULT_TRANSLATIONS` itself ?

Reply:
But is not `DEFAULT_TRANSLATIONS` same as `en/language.json` ? If so, why not use the `.json` file directly as `DEFAULT_TRANSLATIONS` itself ?

Reply:
I need to discuss this point with the team once. We can keep a static default translation JSON in the ABDM application.